test/acceptance/features/bindAppToService.feature

```diff
@@ -585,3 +585,55 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "False"
         And Secret "binding-request-backend-a" contains "BACKEND_READY" key with value "true"
         And Secret "binding-request-backend-a" contains "BACKEND_HOST" key with value "example.common"
+
+
+   Scenario: Backend Service metadata annotations update gets propagated to the binding secret
```

Comment:
so to verify that only changes on service binding annotations trigger reconcile, please add a negative scenario where we show that reconcile is not triggered if non-service binding annotations are updated/added/removed.

Reply:
Updated https://github.com/redhat-developer/service-binding-operator/pull/704/commits/3f8111378c1613385009313ad144a92750b0220f